api/config/swagger.py

```diff
@@ -509,6 +509,25 @@ def schema_view_swagger(request):
                     ],
                     description='Fetching questions on a specific meetup'
                 ),
+                'View a question': coreapi.Link(
+                    url='/api/meetups/{meetupId}/questions/{questionId}/question',
```

Comment:
This `/question` at the end is unnecessary @a-braham. The URL should be `GET /api/meetups/{meetupId}/questions/{questionId}`